lib/coursemology/evaluator/client.rb

```diff
@@ -9,7 +9,11 @@ def self.initialize(host, api_user_email, api_token)
   end
 
   # @param [Boolean] one_shot If the client should only fire one request.
-  def initialize(one_shot = false)
+  # @param [Float] poll_time Sleep time between checks for task allocation.
+  # @param [String] poll_interval Units for poll_time.
+  def initialize(one_shot = false, poll_time = 10, poll_interval = 'seconds')
```

Comment:
It looks like a Rails 5 addition.
http://api.rubyonrails.org/classes/ActiveSupport/Duration.html#method-c-parse